packages/arb-token-bridge-ui/src/hooks/arbTokenBridge.types.ts

```diff
@@ -79,6 +79,8 @@ export type NodeBlockDeadlineStatus =
 export type L2ToL1EventResult = L2ToL1TransactionEvent
 
 export type L2ToL1EventResultPlus = L2ToL1EventResult & {
```

Comment:
Here, and in files below, we add `sender` and/or `destinationAddress` to be able to access them in tx history. Until now we didn't have a need to have them accessible there because always assumed transfers being internal.